components/layout/PaperSection.tsx

```diff
@@ -24,11 +24,11 @@ export default function PaperSection({
 }: PaperSectionProps) {
   return (
     <Paper sx={{ textAlign: 'center', p: 4 }} {...props}>
-      <Typography variant="h2" sx={{ mb: 4 }}>
+      <Typography variant="h2" sx={{ mb: 4 }} id="paper-title">
         {title}
       </Typography>
       {subtitle && (
-        <Typography variant="subtitle1" sx={{ mb: 4 }}>
+        <Typography variant="subtitle1" sx={{ mb: 4 }} id="paper-subtitle">
```

Comment:
Usually, when we use `id`, in general we are trying to garantee that it is unique.

But can you see that those `id`s can be duplicated on our page? If a same page use 2 `PaperSection` we will have duplicated `id`.

Because of that, what do you think to just change `id` to `className`? With that, we 
 will not have problem on that duplicate behavior.

Reply:
ooo that's a great point Leo! I think I'm gonna keep it as `id` just because those elements already have multiple classNames cause of MUI. But I will add a dynamic value to each id so that there are no duplicates. What do you think?

Reply:
Hum, even with that aproach i think we are facing some "problems".

1 - `idSelector` is optional, so if I use `PaperSection` without that prop I will just render something like `paper-title-undefined` as `id` to all `PaperSection` withtou a `idSelector` ( we are facing same duplicated problem, and now we have an `undefined` situtation ).
2 - Passing an extra prop to out component should be our last option, because too many props on a component usually leads to code smell, cause increase component logic size and increase maintainnence efforts.

About the situation of our multiple uses of `classNames`, in my opinion, dont seems to be a great problem. Because the main point of that Task Group is about facilitates debugging, so to me, makes sense that component not to be unique, because we are using it multiple times.

And we can easily debug it by `children`. Which is a pretty good use of Composition Pattern.

In short, I think that passing those `id`s logic seems to be more an "overthinking" about something that we can solve simplier.

Let me know what do you think about

Reply:
you bring up some great points @LeoFuna !
My main thinking with using id's instead of classNames is, personally, I don't see much benefit to adding a className to an element to be used as a selector if it isn't easy to pick it out because that element already has several classNames. 
To address your first point, I don't think there would be a situation where the id would render as `paper-title-undefined` since there is a check to see if that prop has a value before the id is rendered through `idSelector &&`. However I totally agree on your second point and it does seem like I'm over thinking it a bit haha. TBH since in my current implementation in `PaperSection` I'm only ever using idSelector to add id's to the title and subtitle, I think we have 2 solutions:
a) Even though I'm not a fan of this approach, I can add classNames to the title and subtitle for `PaperSection`
b) Since `PaperSection` already has a ton of classNames for its elements, and it's relatively easy to differentiate the title of a `PaperSection` from the subtitle, maybe we should just save us the trouble and not add any additional classNames/id's

Thanks for feedback, let me know what you think!

Reply:
We are having a great conversation about that @sramalho94! Yeah you are right, I missed that `idSelector &&` part sorry :smile: 
But even with that, I still think we will have `id` as `undefined`, because as the first part is not true, will be used the value of the first element (`idSelector`) which is `undefined`.
![image](https://github.com/clearviction-devs/clearviction-wa/assets/80538553/4915b7c6-a4b0-49f3-a590-1d1de2549e99)
I just dont really know what happens with `id` as `undefined` lol, if that `id` will be generated or ignored, but dont seems to be a good approach.

About the solutions that you bring, even tho I still dont think its a huge problem to add another `className`, which we just use for debugging purposes on CV code (styles are not used by className directly), I agree with you that maybe that component dont really needs a new `id` or `className` because its kinda easy to differentiate, and this Task i believe is about that, think on those components and just add `className` or `id` on where is necessary.

So in my opinion we can use your `b` solution. Do you agree with this?

Reply:
@LeoFuna yes this was a great discussion! I agree let's move forward with plan B for this PR, but I think after sleeping on it last night, I'll be more open to using `className` selectors. Sorry about the apprehension haha I just really dislike how some elements have a lot of classNames in the dev tools because of MUI etc. I'll remove the unneeded code from `PaperSection` shortly